.vscode/launch.json

```diff
@@ -8,9 +8,11 @@
             "program": "${workspaceRoot}/packages/pyright/index.js",
             "preLaunchTask": "npm: build:cli:dev",
             "args": [
-                "--verifytypes",
-                "pandas.core.reshape.concat",
+                "--stats",
```

Comment:
is it supposed to be checked in?

Reply:
I don't think it matters as that original one was specific to some other test case.

Reply:
Although maybe that original one runs on everybody's machine. I can revert it.